packages/vscode-boxel-tools/README.md

```diff
@@ -17,11 +10,66 @@ You can also build it [locally](#development).
 
 ## Usage
 
-Open a new window and in the command palette select "Boxel Tools: Attach to Boxel Workspaces".
+### Getting Started
+
+1. Install the extension from the VS Code Marketplace or [build it locally](#development).
+2. Open a new window and in the command palette (F1 or Ctrl+Shift+P), select "Boxel Tools: Attach to Boxel Workspaces".
+3. You will be prompted to log in with your Matrix credentials:
+   - Enter your username and password
+   - If you want to use a different Matrix server, change the server URL in VS Code settings
+
+### Connecting to Boxel Workspaces
+
+After logging in, the extension will automatically discover and list all your Boxel workspaces in the "Boxel" view in the Explorer sidebar. For each workspace, you can:
+
+- **Pull workspace files**: Click the download icon next to a workspace to download all files to your local machine
+- **Push local changes**: Click the upload icon to push any local changes back to the remote workspace
+- **Enable file watching**: Toggle automatic syncing of changes by clicking the "eye" icon
+
+To add the Boxel workspaces to your VSCode workspace, use the + button in the Boxel view or the command "Add Boxel Workspaces Folder to Workspace".
+
+### Working with Workspace Files
+
+Once you've downloaded workspace files:
+
+1. Navigate to the files in your Explorer view
+2. Edit files as you normally would in VS Code
+3. If file watching is enabled, changes will automatically sync to the remote workspace
```

Comment:
Is the opposite true? Will changes on the server automatically sync to the local workspace? We should probably be clear about this in the docs.

Reply:
It's not, no - only pushing changes live. We should make this clear both in the docs and also probably rename the part of the extension that does this so it's clear that this is the direction. "Auto push changes" or something.

We have a matrix client in this, so with moving sse to matrix events we could do bidirectional live syncing probably, although being clear about how conflicts are handled is important.